### PR TITLE
Accessibility issue for form

### PR DESCRIPTION
**Pull request by seanchoi-dev**
Issue ticket: [MWPW-123053](https://jira.corp.adobe.com/browse/MWPW-123053)

Before: https://main--pages--adobe.hlx.page/general/en/blog-survey22/
After: https://accessibility-issue--pages--adobe.hlx.page/general/en/blog-survey22/

### CODE REVIEW

File: pages/blocks/form/form.js [hunk 1]
```diff
@@ -388,7 +388,7 @@ function createForm({
       formField += `
         <div class="input-el question ${required ? 'is-required' : ''}">
           <div class="title-el">
-            <label class="label-title" for="${name}">${setup.label}</label>
+            <label class="label-title" tabindex="0" for="${name}">${setup.label}</label>
```
Comment (awkawk) [inline pages/blocks/form/form.js:391]: Why putting a tabindex on a label?

Comment (awkawk) [reply pages/blocks/form/form.js:391]: (It isn't expected to be focusable)

Comment (awkawk) [reply pages/blocks/form/form.js:391]: I recommend leaving all tabindex="0" out - this isn't recommended. It actually creates a different problem when users expect that a focusable item is interactive, but the legends are not. The assistive technology will use the semantic association to read the groups correctly without the legend be focusable.

Comment (seanchoi-dev) [reply pages/blocks/form/form.js]: I've removed them.

Comment (seanchoi-dev) [reply pages/blocks/form/form.js]: FYI, I added them yesterday and saw your comment today from ticket to add the fieldset/legends. Forgot to remove them. Good catch.

**Comment by awkawk** [comment]
Looks great for the fixes - the one remaining from the bug is the aria-label for the banner/heading link.

**Comment by seanchoi-dev** [comment]
> Looks great for the fixes - the one remaining from the bug is the aria-label for the banner/heading link.

@awkawk Just made another commit to add aria-label for the icons.

**Comment by awkawk** [comment]
Great - all good based on my review. Thanks.

**Review by awkawk — APPROVED** [review]

**Pull request merged**